jib-gradle-plugin/src/test/resources/gradle/projects/all-local-multi-service/lib/src/test/java/com/lib/LibTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.lib;
+
+import org.junit.Assert;
+import org.junit.Test;
```

Comment:
Q: how is it possible that this class is compiled? I don't see a junit dep defined in `lib/build.gradle`.

Reply:
Hah that's a good question, no actual test is run there in the end. The dependency doesn't trigger running tests. I think I'll just take this class out. But it should have no effect.